QUEST.tsv

```diff
@@ -497,22 +497,22 @@ QUEST_20150317_000496	What makes you walk towards the light in a place full of d
 QUEST_20150317_000497	The harder you try to find it, the stronger you become, but if you don't find it, you weaken. {nl}What is it?
 QUEST_20150317_000498	Aide de Camp Selina
 QUEST_20150317_000499	The number of days where we have to stay self-sufficient is growing. {nl}We sometimes joke about plowing a farm for ourselves, {nl}but nowadays we're actually considering it.
-QUEST_20150317_000500	Watcher
+QUEST_20150317_000500	Beholder
 QUEST_20150317_000501	$If it isn't you, the one who made me release Goddess Saule. {nl}What are you putting your life on the line for?
 QUEST_20150317_000502	There's no need for you to work anymore. {nl}You should follow the others' example and immerse yourself in peace, {nl}and pray the Goddess will grant you your wishes.
-QUEST_20150317_000503	I warned you but you just wouldn't listen.
-QUEST_20150317_000504	You shall pay for your choice.
-QUEST_20150317_000505	I underestimated you.
+QUEST_20150317_000503	I warned you, but you just wouldn't listen.
+QUEST_20150317_000504	You shall pay the price for your choice.
+QUEST_20150317_000505	I have underestimated you.
 QUEST_20150317_000506	{memo X}This isn't over! {nl}I'll be watching from the Fallen Worry Forest.
 QUEST_20150317_000507	Warning
 QUEST_20150317_000508	Polluted. Do not use!
-QUEST_20150317_000509	This is the first revelation I made. {nl}For you, it would be the third.
-QUEST_20150317_000510	My strength will weaken every time I put a part of myself in the revelation. {nl}By the time this revelation reaches you, more than a thousand years will have passed, {nl}and not even my consciousness would exist.
-QUEST_20150317_000511	That is why I needed to find a guardian to protect the revelation. {nl}Unfortunately, my sisters and I did not come to an agreement {nl}and were not able to leave the revelation to anyone.
-QUEST_20150317_000512	Only Saule followed me so I could leave this revelation. {nl}And then for a very long time, I went on a journey in search of a guardian.
-QUEST_20150317_000513	When I thought about humans like you, I saw the light. {nl}The demons look down on humans so {nl}they could not even imagine that I would leave the revelation to you.
-QUEST_20150317_000514	But most of all, I trusted people. {nl}You must have seen it yourself by now. {nl}Despite going through disasters such as the Tree of Despair, {nl}people don't lose hope easily.
-QUEST_20150317_000515	The guardian of the next revelation will be one who unites the kingdom. {nl}You can find my next revelation in the deepest part of the mausoleum where he rests.
+QUEST_20150317_000509	This is the first Revelation I made.{nl}For you, it would be the third.
+QUEST_20150317_000510	My strength will weaken every time I put a part of myself in the Revelation.{nl}By the time this Revelation reaches you, more than a thousand years will have passed, {nl}and not even my consciousness would exist.
```

Comment:
Shouldn't it be "(...) every time I put a part of myself into a Revelation."? There are multiple revelations, and with each one she makes, she weakens.